content/docs/test.md

```diff
@@ -0,0 +1,329 @@
+
+## Images
+
+To format an image, use similar syntax to [links](#links), but add a leading `!`
+character. The square brackets contain the image's alt text. Try to always use
+alt text so that people using screen readers can get some benefit from the
+image.
+
+![pencil icon](/img/pencil.png)
+
+To specify extended attributes, such as width, title, caption, etc, use the
+<a href="https://gohugo.io/content-management/shortcodes/#figure">figure shortcode</a>,
+which is preferred to using a HTML `<img>` tag. Also, if you need the image to
+also be a hyperlink, use the `link` attribute, rather than wrapping the whole
+figure in Markdown link syntax as shown below.
+
+{{< figure src="/img/pencil.png" title="Pencil icon" caption="Image used to illustrate the figure shortcode" width="200px" >}}
+
+Even if you choose not to use the figure shortcode, an image can also be a link. This
+time the pencil icon links to the Kubernetes website. Outer square brackets enclose
+the entire image tag, and the link target is in the parentheses at the end.
+
+[![pencil icon](/img/pencil.png)](https://kubernetes.io)
+
+## Tables
+
+Simple tables have one row per line, and columns are separated by `|`
+characters. The header is separated from the body by cells containing nothing
+but at least three `-` characters. For ease of maintenance, try to keep all the
+cell separators even, even if you heed to use extra space.
+
+| Heading cell 1 | Heading cell 2 |
+|----------------|----------------|
+| Body cell 1    | Body cell 2    |
+
+The header is optional. Any text separated by `|` will render as a table.
+
+Markdown tables have a hard time with block-level elements within cells, such as
+list items, code blocks, or multiple paragraphs. For complex or very wide
+tables, use HTML instead.
+
+<table>
+<thead>
+  <tr>
+    <th>Heading cell 1</th>
+    <th>Heading cell 2</th>
+  </tr>
+</thead>
+<tbody>
+  <tr>
+    <td>Body cell 1</td>
+    <td>Body cell 2</td>
+  </tr>
+</tbody>
+</table>
+
+## Sidebars and Admonitions
+
+Sidebars and admonitions provide ways to add visual importance to text. Use
+them sparingly.
+
+### Sidebars
+
+A sidebar offsets text visually, but without the visual prominence of
+[admonitions](#admonitions).
+
+> This is a sidebar.
+>
+> You can have paragraphs and block-level elements within a sidebar.
+>
+> You can even have code blocks.
+>
+> ```bash
+> sudo dmesg
+> ```
+>
+
+### Admonitions
+
+Admonitions (notes, warnings, etc) use Hugo shortcodes.
+
+{{< note >}}
+Notes catch the reader's attention without a sense of urgency.
+
+You can have multiple paragraphs and block-level elements inside an admonition.
+
+| Or | a | table |
+| :-- | --- | --- |
+{{< /note >}}
+
+{{< caution >}}
+The reader should proceed with caution.
+{{< /caution >}}
+
+
+{{< warning >}}
+Warnings point out something that could cause harm if ignored.
+{{< /warning >}}
+
+## Katacoda Embedded Live Environment
+
+{{< kat-button >}}
```

Comment:
```suggestion
```

Reply:
No to `Katacoda`? I was super excited to see it, it's really nifty tool... I _suppose_ we don't need it here...